falcon/response.py

```diff
@@ -746,6 +749,10 @@ def add_link(self, target, rel, title=None, title_star=None,
         if anchor is not None:
             value += '; anchor="' + uri_encode(anchor) + '"'
 
+        if crossorigin is not None:
+            if crossorigin in ['anonymous', 'use-credentials']:
```

Comment:
Rather than failing silently when the value is invalid, I think we ought to raise an instance of ValueError in that case.